### ISSUE TITLE
Stuck "Open Orders" in Daily view

### ISSUE THREAD

**Issue by AnthonyHunt** [leihs/leihs#925]
A strange hang up from 4.19 through 5.2.2...

We have 6 phantom open orders in our daily view. There is nothing pending behind the button though?

<img width="193" alt="Screenshot 2020-10-08 at 16 31 28" src="https://user-images.githubusercontent.com/299919/95480510-e3ac0780-0983-11eb-88a8-b851cf4a2100.png">
<img width="526" alt="Screenshot 2020-10-08 at 16 31 59" src="https://user-images.githubusercontent.com/299919/95480534-e870bb80-0983-11eb-8021-31fe22430684.png">

These appeared at some point in 4.18? 4.19? At first there was 1, then another, then another. I can't see why they are there? I believe they started to appear around the time the new countdown code was implemented? We have lived with them but now they are getting annoying.

Your thoughts on removing them? I have checked everywhere I can think of - maybe this might need a database fix?

**Comment by AnthonyHunt-UoW** [comment]
Could this be related to https://github.com/leihs/leihs/issues/1147? Where a reservation was never completed? or had lines removed but left a record that couldn't be deleted properly?

**Comment by nimaai** [comment]
Could you check/uncheck "No verification required" and see if the List of Orders changes and you see the "phantom" orders?

**Comment by AnthonyHunt-UoW** [comment]
Yes, that was my first thought. I've also changed the date manually back to 10 years ago in case there was a date filter being inadvertently applied.

**Comment by nimaai** [comment]
then you could try:

```sql
SELECT orders.*, reservations.id, reservations.status
FROM orders
LEFT JOIN reservations ON reservations.order_id = orders.id
WHERE orders.state = 'submitted'
  AND orders.inventory_pool_id = '<YOUR POOL ID>'
ORDER BY orders.id
```

**Comment by AnthonyHunt-UoW** [comment]
That returned the 6 records....

                 id                  |               user_id                |          inventory_pool_id           |             purpose             |   state   |         created_at         |         updated_at         | reject_reason |          customer_order_id           | lending_terms_accepted | id | status 
--------------------------------------+--------------------------------------+--------------------------------------+---------------------------------+-----------+----------------------------+----------------------------+---------------+--------------------------------------+------------------------+----+--------
 7ffc1140-cc4b-4d36-95f0-f99fb923f289 | fee7e9af-d85f-52a0-93ca-47847c93fad6 | 040f275a-cbc2-5c21-9e98-8613fde1ccab | JOUR1009 practical work         | submitted | 2019-01-30 09:31:51.244742 | 2021-09-08 16:03:25.391474 |               | e447c464-ea2f-47ab-b9e3-49b31548a3db |                        |    | 
 b846c41d-cb05-4eb9-bb08-042fac262725 | 5870a47b-2cb9-544f-b8b8-106ad082adb0 | 040f275a-cbc2-5c21-9e98-8613fde1ccab | DFPR                            | submitted | 2020-01-24 14:20:09.215198 | 2021-09-08 16:03:28.33625  |               | 5de6f5f9-bd96-4d2d-a17c-48a54388aadf |                        |    | 
 be9330b6-9216-4c12-89a2-8dd5540bd334 | 51c16271-bd5c-4154-94e7-f782ef1ba36d | 040f275a-cbc2-5c21-9e98-8613fde1ccab | JOUR1009- Doing a vox interview | submitted | 2019-09-30 09:53:09.646045 | 2021-09-08 16:03:27.150354 |               | fad83f84-7faf-4201-a989-257550e5c7f5 |                        |    | 
 ca9759d0-dfb6-47c7-b9f3-e6c1beb14937 | 4226b29d-3f22-5d96-876b-381f3391baea | 040f275a-cbc2-5c21-9e98-8613fde1ccab | DFPR2010E1                      | submitted | 2019-10-01 13:38:39.354705 | 2021-09-08 16:03:27.244958 |               | 6fb2d571-b651-409d-91d6-aa30f76d0372 |                        |    | 
 cc238f0f-8f61-407d-9aea-2249ed1b2b8d | 46b3719d-d7f7-59a6-a229-5bb881d7450e | 040f275a-cbc2-5c21-9e98-8613fde1ccab | CDME                            | submitted | 2019-01-31 14:32:14.890603 | 2021-09-08 16:03:25.420345 |               | 1044d3e9-b5f0-49f6-9857-1e188cbe8ef7 |                        |    | 
 e3a32172-8b92-4889-bca9-025959209fc6 | 5c7196c2-62f4-57e8-8d80-e8b8a1d2c694 | 040f275a-cbc2-5c21-9e98-8613fde1ccab | CMED1003                        | submitted | 2019-03-21 20:33:24.587779 | 2021-09-08 16:03:26.232004 |               | b88d1236-db52-4536-9e1f-cc5068491056 |                        |    |

**Comment by AnthonyHunt-UoW** [comment]
It appears that these records don't have an entry in the table "reservations"? but do in the "orders" table. Confirmed by some clunky SQL...

SELECT reservations.*
FROM reservations
WHERE reservations.status <> 'submitted' AND reservations.status <> 'closed' AND reservations.status <> 'signed' AND reservations.status <> 'approved' AND reservations.status <> 'rejected' AND reservations.status <> 'unsubmitted';
